mountpoint-s3/src/cli.rs

```diff
@@ -237,21 +238,19 @@ pub struct CliArgs {
 
     #[clap(
         long,
-        help = "Enable caching of object metadata and content to the given directory",
+        help = "Enable caching of object content to the given directory and set metadata TTL to 60s",
         help_heading = CACHING_OPTIONS_HEADER,
         value_name = "DIRECTORY",
     )]
     pub cache: Option<PathBuf>,
 
     #[clap(
         long,
-        help = "Time-to-live (TTL) for cached metadata in seconds [default: 1s]",
-        value_name = "SECONDS",
-        value_parser = parse_ttl_seconds,
+        help = "Time-to-live (TTL) for cached metadata in seconds [default: minimal, or 60s if --cache is set]",
```

Comment:
nit to not make people think `60s` is a valid value to pass:
```suggestion
        help = "Time-to-live (TTL) for cached metadata in seconds [default: minimal, or 60 seconds if --cache is set]",
```